okteto.yml

```diff
@@ -20,3 +20,16 @@ dev:
       - /go/pkg/
       - /root/.cache/go-build/
     autocreate: true
+test:
+  unit:
+    image: okteto/golang:1
+    artifacts:
+      - coverage.txt
+      - coverage.html
+    caches:
+      - /go/pkg/
+      - /root/.cache/go-build/
+    commands:
+      - mkdir -p /var/lib/dbus && echo "11111111111111111111111111111111" | tee /var/lib/dbus/machine-id && chmod 644 /var/lib/dbus/machine-id
+      - git init && git remote add origin https://github.com/okteto/okteto.git
```

Comment:
Why are these 2 lines needed? I assume is because unit test need them for some reason. 

If this is needed, it is probably because our unit tests are not so "unit" as we believe, so we might need to take a loot at it at some point (not as part of this PR). But could we at least add a comment on each line indicating why is needed? In that way, we don't forget about it or take it into account when working with the tests